ApollosProject/apollos-apps
# chore: Remove Community Placeholder Copy

## ✏️ Description

There was a bit of placeholder copy that made it into the Community feature and this PR removes it

## 🔬 To Test
1. Open up the Community Tab
2. Make sure you're a part of the Community
3. Check out the Community Tab

## 📸 Screenshots

| Before | After |
| --- | --- |
| ![CleanShot_2023 02 28__000015](https://user-images.githubusercontent.com/45076058/221989414-92d77c2c-ccd3-4402-9672-dbf7a08f7c3b.png) | ![CleanShot_2023 02 28__000016](https://user-images.githubusercontent.com/45076058/221989493-f1fcddd9-1247-4e84-b5f3-b24a51c21db3.png) |



## ⚠️ To-do Before Merge

n/a

## 📝 Checklist
